safe-svg.php

```diff
@@ -142,7 +142,7 @@ public function __construct() {
 			add_filter( 'wp_get_attachment_image_src', array( $this, 'one_pixel_fix' ), 10, 4 );
 			add_filter( 'admin_post_thumbnail_html', array( $this, 'featured_image_fix' ), 10, 3 );
 			add_action( 'admin_enqueue_scripts', array( $this, 'load_custom_admin_style' ) );
-			add_action( 'get_image_tag', array( $this, 'get_image_tag_override' ), 10, 6 );
+			add_filter( 'get_image_tag', array( $this, 'get_image_tag_override' ), 10, 6 );
```

Comment:
Nice catch! Technically behind the scenes, WordPress converts all `add_action` calls to `add_filter` calls so this shouldn't cause any issues but still nice to have this be the correct hook type